src/common/websocket/api.calls.types.ts

```diff
@@ -0,0 +1,802 @@
+        response: CountriesListResponse
+    }
+    revoke_oauth_app: {
+        request: RevokeOauthApplicationRequest
+        response: RevokeOauthApplicationResponse
+    }
+    sell_contract_for_multiple_accounts: {
+        request: SellContractsMultipleAccountsRequest
+        response: SellContractsMultipleAccountsResponse
+    }
+    sell_expired: {
+        request: SellExpiredContractsRequest
+        response: SellExpiredContractsResponse
+    }
+    sell: {
+        request: SellContractRequest
+        response: SellContractResponse
+    }
+    set_account_currency: {
+        request: SetAccountCurrencyRequest
+        response: SetAccountCurrencyResponse
+    }
+    set_financial_assessment: {
+        request: SetFinancialAssessmentRequest
+        response: SetFinancialAssessmentResponse
+    }
+    set_self_exclusion: {
+        request: SetSelfExclusionRequest
+        response: SetSelfExclusionResponse
+    }
+    set_settings: {
+        request: SetAccountSettingsRequest
+        response: SetAccountSettingsResponse
+    }
+    statement: {
+        request: StatementRequest
+        response: StatementResponse
+    }
+    states_list: {
+        request: StatesListRequest
+        response: StatesListResponse
+    }
+    ticks_history: {
+        request: TicksHistoryRequest
+        response: TicksHistoryResponse
+    }
+    ticks: {
+        request: TicksStreamRequest
+        response: TicksStreamResponse
+    }
+    time: {
+        request: ServerTimeRequest
+        response: ServerTimeResponse
+    }
+    tnc_approval: {
+        request: TermsAndConditionsApprovalRequest
+        response: TermsAndConditionsApprovalResponse
+    }
+    topup_virtual: {
+        request: TopUpVirtualMoneyAccountRequest
+        response: TopUpVirtualMoneyAccountResponse
+    }
+    trading_durations: {
+        request: TradingDurationsRequest
+        response: TradingDurationsResponse
+    }
+    trading_platform_investor_password_reset: {
+        request: TradingPlatformInvestorPasswordResetRequest
+        response: TradingPlatformInvestorPasswordResetResponse
+    }
+    trading_platform_password_reset: {
+        request: TradingPlatformPasswordResetRequest
+        response: TradingPlatformPasswordResetResponse
+    }
+    trading_servers: {
+        request: ServerListRequest
+        response: ServerListResponse
+    }
+    trading_times: {
+        request: TradingTimesRequest
+        response: TradingTimesResponse
+    }
+    transaction: {
+        request: TransactionsStreamRequest
+        response: TransactionsStreamResponse
+    }
+    transfer_between_accounts: {
+        request: TransferBetweenAccountsRequest
+        response: TransferBetweenAccountsResponse
+    }
+    unsubscribe_email: {
+        request: UnsubscribeEmailRequest
+        response: UnsubscribeEmailResponse
+    }
+    // The types for the verify_email_cellxpert is not correct in @deriv/api-types, so I'm using the correct one here
```

Comment:
@mohsen-deriv This is fixed, You just need to update `@deriv/api-types` package.

Reply:
fixed it thanks :pray: